apps/src/lib/ui/lti/link/LtiLinkAccountPage/cards/ContinueCard/index.tsx

```diff
@@ -57,19 +57,21 @@ const ContinueAccountCard = () => {
         title={i18n.ltiLinkAccountNewAccountCardHeaderLabel()}
         icon={
           <FontAwesomeV6Icon
-            className={classNames(styles.icon, 'fa-3x')}
+            className={classNames(styles.icon, 'fa-2x')}
             iconName={'user-plus'}
           />
         }
       />
-      <CardContent>
-        {i18n.ltiLinkAccountContinueAccountCardContent()}
-      </CardContent>
+      <div className={classNames(styles.cardContentContainer)}>
+        <CardContent>
+          {i18n.ltiLinkAccountContinueAccountCardContent()}
+        </CardContent>
+      </div>
```

Comment:
I haven't tested this, but I believe you can have the `cardContentContainer` class be passed into `CardContent` directly and avoid the parent div.

```suggestion
      <CardContent className={classNames(styles.cardContentContainer)}>
         {i18n.ltiLinkAccountContinueAccountCardContent()}
      </CardContent>
```

Reply:
Oh I had tried to pass them in but ran into some typescript errors (though I'm new to typescript so I may have just been implementing wrong)!

Reply:
I can definitely dig deeper but want to hear back about what approach we'll take regarding the generalization vs. cloning discussion so I'll hold off on further updates until that's figured out.